fhir-smart/src/test/java/org/linuxforhealth/fhir/smart/test/AuthzPolicyEnforcementTest.java

```diff
@@ -353,6 +353,39 @@ public void testDirectPatientInteraction() throws FHIRPersistenceInterceptorExce
         }
     }
 
+    @Test
+    public void testUserInteraction() throws FHIRPersistenceInterceptorException {
+        // don't use buildRequestHeaders because we want to test without a "patient_id" claim
+        Map<String, List<String>> requestHeaders = new LinkedHashMap<String, List<String>>();
+        List<String> authHeader = Collections.singletonList("Bearer " + JWT.create()
+                .withClaim("scope", "user/Patient.read")
+                .sign(Algorithm.none()));
+        requestHeaders.put("Authorization", authHeader);
+        FHIRRequestContext.get().setHttpHeaders(requestHeaders);
+
+        try {
+            properties.put(FHIRPersistenceEvent.PROPNAME_RESOURCE_TYPE, "Patient");
+            properties.put(FHIRPersistenceEvent.PROPNAME_RESOURCE_ID, PATIENT_ID);
+            FHIRPersistenceEvent event = new FHIRPersistenceEvent(patient, properties);
+            interceptor.beforeRead(event);
+            interceptor.beforeVread(event);
+            interceptor.beforeHistory(event);
+        } catch (FHIRPersistenceInterceptorException e) {
+            fail("Patient interaction was not allowed but should have been", e);
+        }
+
+        try {
+            properties.put(FHIRPersistenceEvent.PROPNAME_RESOURCE_TYPE, "Patient");
+            properties.put(FHIRPersistenceEvent.PROPNAME_RESOURCE_ID, PATIENT_ID);
+            FHIRPersistenceEvent event = new FHIRPersistenceEvent(patient, properties);
+            interceptor.beforeRead(event);
+            interceptor.beforeVread(event);
+            interceptor.beforeHistory(event);
```

Comment:
Need to fail here if no exception

Reply:
fixed line 378 to use a different reosurce type and added the fail call